app/src/main/res/layout/fragment_create_account.xml

```diff
@@ -1,19 +1,168 @@
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/sign_up_password"
+        android:scaleType="centerCrop"
+        android:src="@drawable/maingirl"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/logo"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="8dp"
-        android:layout_marginTop="8dp"
-        android:layout_marginEnd="8dp"
-        android:textAlignment="center"
-        android:textSize="20sp"
+        android:layout_gravity=""
+        android:contentDescription="@string/app_name"
+        android:src="@drawable/logo"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        tools:layout_editor_absoluteX="207dp"
+        tools:layout_editor_absoluteY="437dp">
+
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="0.6">
+
+            <EditText
+                android:id="@+id/CreateAccount"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:textAlignment="center"
+                android:ems="10"
+                android:inputType="textPersonName"
+                android:text="@string/sign_up_create_account"
+                android:textColor="@color/colorAccent"
+                android:textSize="@dimen/create_account_text_size"
+                android:importantForAutofill="no" />
+        </FrameLayout>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="0.4"
+
+
+            android:orientation="vertical">
+
+
+            <EditText
+                android:id="@+id/UserName"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:layout_marginStart="20dp"
+                android:layout_marginEnd="20dp"
+                android:layout_marginBottom="10dp"
+                android:autofillHints=""
+                android:background="@drawable/edittext_background"
+                android:hint="@string/sign_up_username"
+                android:inputType="textPersonName"
+                android:padding="10dp"
+                tools:targetApi="o" />
+
+            <EditText
```

Comment:
This EditText only allows users to input numbers. Please change this to a password EditText